docs/experiments-covid.md

```diff
@@ -1,240 +1,182 @@
-# Working with the [COVID-19 Open Research Dataset](https://pages.semanticscholar.org/coronavirus-research)
+# TREC-COVID Baselines
 
-This document describes various tools for working with the [COVID-19 Open Research Dataset (CORD-19)](https://pages.semanticscholar.org/coronavirus-research) from the [Allen Institute for AI](https://allenai.org/).
-For an easy way to get started, check out our Colab demos, also available [here](https://github.com/castorini/anserini-notebooks):
+This document describes various baselines for the [TREC-COVID Challenge](https://ir.nist.gov/covidSubmit/), which uses the [COVID-19 Open Research Dataset (CORD-19)](https://pages.semanticscholar.org/coronavirus-research) from the [Allen Institute for AI](https://allenai.org/).
+Here, we focus on running retrieval experiments; for basic instructions on building Anserini indexes, see [this page](experiments-cord19.md).
 
-+ [Colab demo using the title + abstract index](https://github.com/castorini/anserini-notebooks/blob/master/pyserini_covid19_default.ipynb)
-+ [Colab demo using the paragraph index](https://github.com/castorini/anserini-notebooks/blob/master/pyserini_covid19_paragraph.ipynb)
-+ [Colab demo that demonstrates integration with SciBERT](https://github.com/castorini/anserini-notebooks/blob/master/Pyserini+SciBERT_on_COVID_19_Demo.ipynb)
+## Round 1 Runs
 
-We provide instructions on how to build Lucene indexes for the collection using Anserini below, but if you don't want to bother building the indexes yourself, we have pre-built indexes that you can directly download:
+tl;dr - here are the runs that can be easily replicated with Anserini:
 
-If you don't want to build the index yourself, you can download the latest pre-built copies here:
+|    | index     | field(s)                 | ndcg@10 |
```

Comment:
For the sake of completion, maybe add MAP and P@N?

Reply:
I don't want to add MAP because we have too many unjudged docs... and P@N is largely going to correlation - I want to focus on the tl;dr and not distract?

Reply:
Sure, sounds good!

Reply:
Agree with MAP. P@7 might be nice because every assessed run with have complete information.